Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -258,17 +263,55 @@ class SmartPage_Context
     }
   };
 
-  public SmartPage_PageRequestManager_endRequest = function (sender: unknown, args: Sys.WebForms.EndRequestEventArgs)
+  public SmartPage_PageRequestManager_endRequest = (sender: unknown, args: Sys.WebForms.EndRequestEventArgs) =>
```

Comment:
We discussed stripping out the prefix: 
`Sys.WebForms.PageRequestManagerServerErrorException:` before rendering the message. that's still missing.